### DISCUSSION TITLE
Ant upload being freeze when select application file on Mac

### DISCUSSION THREAD

**Discussion by sakamiq1** [ant-design/ant-design#45230] [Q&A]
Hi, i'm currently using ant upload for my multiple files uploading
This is the problem that i occurred, please watch the video that i share (it some how happen on my Mac but other laptop not)
I tried other library but seem like they have the same issue, please explain to me, thanks
check this link bellow
https://drive.google.com/file/d/1ChaiukU_l7G2T2W5il-Q2Lbzo3_PDnVC/view?usp=sharing

**Comment by yoyo837** [comment]
You can try whether the html native `<input type="file" />` tag also has this problem at first.
